app/views/enrollments/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="container-fluid text-center">
+  <br>
+  <div class="row">
+    <div class="col-2"></div>
+    <div class="col-8">
+      <div class="w-50 bg-dark mx-auto text-light">
+        <%=flash[:notice]%>
+      </div>
+      <br>
+      <table class="table table-hover table-bordered">
+        <thead>
+          <tr>
+            <th colspan="6"><h3>List of Events</h3></th>
+          </tr>
+          <tr>
+            <th>#</th>
+            <th>Name</th>
+            <th>Description</th>
+            <th>Category</th>
+            <th>Event Date</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% n = 1 %>
+          <% @events.each do |event| %>
```

Comment:
why used n = 1, what is requirement of it?

Reply:
It used to give an index numbers to the entries.